### DISCUSSION TITLE
Request for English Translation and Alternative Payment method for international fans/supporters

### DISCUSSION THREAD

**Discussion by muhammadammarzahid** [MuiseDestiny/zotero-gpt#366] [General]
I am using your plugins and I am absolutely enjoying them. Although the instructions are in Chinese, but I can make sense by using Google Translate. It would be grate if you can translate the plugins i English to grow the audience. I can help you with the translation (I am not technical enough though to tinker with the code). Secondly, the payment method for the Pro versions is not available in my country (Qatar). I can not even install the payment app on my phone. An alternative method for international fans would be a great addition.

**Comment by LHSequalsRHS** [comment]
I second this. There is no proper English documentation. The tool is functional, but poor documentation, especially in English, will fail to attract users.

**Comment by edvinPL** [comment]
Bump!
Such a powerful tool, so many options.
But very difficult to use without knowing chinese and no localization. Trying to translate as much as possible with AI - though supporting you or buying PRO is impossible without being in China (?)

**Comment by Dark-Obsessions** [comment]
Can you provide an English version and an international payment method, please? I want to help support this awesome and very useful project. thanks in advance!
